msteams-platform/task-modules-and-cards/cards/cards-reference.md

```diff
@@ -70,7 +70,8 @@ A customizable card that can contain any combination of text, speech, images, bu
 |
 
 > [!NOTE]
-> Media elements are currently not supported in Adaptive cards v1.2 on the Teams platform.
+> * Media elements are currently not supported in Adaptive cards v1.2 on the Teams platform.
+> * Teams platform supports up to version 1.2 of Adaptive card features.
 
```

Comment:
MSTP does not support usage of 'up to'. Use earlier/later instead.
```Suggestion
> * Teams platform supports v1.2 or earlier of Adaptive card features.
> * Media elements are currently not supported in Adaptive card v1.2 on the Teams platform.
```
Please verify the above and update the note.